src/System.ComponentModel.TypeConverter.TestData/System.ComponentModel.TypeConverter.TestData.csproj

```diff
@@ -1,7 +1,2 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <PackageVersion>1.0.4</PackageVersion>

```

Comment:
What’s the package version now going to be?

Reply:
it's going to 5.0.0-... something. Either alpha or beta, whatever the default is right now. Will figure this out when I do the internal builds.